src/applications/personalization/preferences/helpers.jsx

```diff
@@ -189,6 +189,11 @@ export const benefitChoices = [
     gaTag: 'Family',
     introduction:
       'If you’re the family member of a Veteran or Servicemember, you may qualify for benefits yourself. If you’re a caregiver for a Veteran with service-connected disabilities, you may qualify for additional benefits and support for yourself and the Veteran you’re caring for.',
+    cta: {
+      link: '/family-member-benefits/',
+      text: 'View all related benefits',
+      gaTag: 'View all related benefits',
```

Comment:
Should this be kebab-case?

Reply:
The case is the same as the other pre-existing blocks -- I just copied an existing cta block and pasted it into the Family block for this new CTA button.
That said, I can definitely do a file-global find-n-replace if you think the case should be different.
Feel free to either Resolve the convo (let it be) or Comment further (change all key-instances).

Reply:
which `cta` block did you copy? I see a bunch of `cta.gaTag`s being kebab-case (line 176, 138, 117, etc)

Reply:
Strange!
Can't remember which one I copied from, BUT I just double-checked using this PR's View File (/department-of-veterans-affairs/vets-website/blob/a100693cf5f8f97f56d50356fd4afd5fc9e9190f/src/applications/personalization/preferences/helpers.jsx) and I'm not seeing what you saw -- all the GA Tag keys are 'gaTag' there.

Reply:
LOL, total miscommunication here. I meant the _value_ of the `gaTag` should be `view-all-related-benefits`

Reply:
ROFL!  I'll kebab the value.
Leaving this convo unresolved 'til my fix is pushed up.

Reply:
gaTag value kebab-case fix pushed up.